nautobot/utilities/forms/fields.py

```diff
@@ -458,3 +458,92 @@ def prepare_value(self, value):
         if value is None:
             return ""
         return json.dumps(value, sort_keys=True, indent=4)
+
+
+class JSONArrayFormField(forms.JSONField):
+    """
+    A FormField counterpart to JSONArrayField.
+    Replicates ArrayFormField's base field validation: Field values are validated as JSON Arrays,
+    and each Array element is validated by |base_field| validators.
+    """
+
+    def __init__(self, base_field, *, delimiter=",", **kwargs):
+        self.base_field = base_field
+        self.delimiter = delimiter
+        super().__init__(**kwargs)
+
+    def clean(self, value):
+        """
+        Validate |value| and return its "cleaned" value as an appropriate
+        Python object. Raise ValidationError for any errors.
+        """
+        value = super().clean(value)
+        return [self.base_field.clean(val) for val in value]
+
+    def prepare_value(self, value):
+        """
+        Return a string of this value.
+        """
+        if isinstance(value, list):
+            return self.delimiter.join(str(self.base_field.prepare_value(v)) for v in value)
+        return value
+
+    def to_python(self, value):
+        """
+        Convert |value| into JSON, raising django.core.exceptions.ValidationError
+        if the data can't be converted. Return the converted value.
+        """
+        if isinstance(value, list):
+            items = value
+        elif value:
+            items = value.split(self.delimiter)
```

Comment:
Do we need to catch a potential `AttributeError` and/or `TypeError` here and map them into a `ValidationError`?